tests/qeidentity/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_subdirectory(host)
+
+if (UNIX)
+	add_subdirectory(enc)
+endif()
+
+# Run all tests. Also generate a qeidentity.
```

Comment:
Also generate a qeidentity is not applicable here.

The report test saves a report to disk so that another tests can load the report and attest it.

Reply:
removed comment